src/ast/query.rs

```diff
@@ -1671,6 +1679,81 @@ impl fmt::Display for OrderByExpr {
             Some(false) => write!(f, " NULLS LAST")?,
             None => (),
         }
+        if let Some(ref with_fill) = self.with_fill {
+            write!(f, " {}", with_fill)?
+        }
+        if let Some(ref interpolate) = self.interpolate {
+            match &interpolate.expr {
+                Some(exprs) => write!(
+                    f,
+                    " INTERPOLATE ({})",
```

Comment:
Is it better to use `display_comma_separated` here?

Reply:
Yep. Good call. I forgot this existed. Thank you. Fixed in [3e8b141](https://github.com/sqlparser-rs/sqlparser-rs/pull/1330/commits/3e8b141b7a19f8c16c535c62c17e61a3400e2dd1)